packages/clerk-js/src/ui/components/OrganizationList/OrganizationListPage.tsx

```diff
@@ -77,46 +80,30 @@ const CreateOrganizationButton = ({
 };
 
 export const OrganizationListPage = withCardStateProvider(() => {
-  const card = useCardState();
-  const { userMemberships, userSuggestions, userInvitations } = useOrganizationListInView();
-  const isLoading = userMemberships?.isLoading || userInvitations?.isLoading || userSuggestions?.isLoading;
-  const hasAnyData = !!(userMemberships?.count || userInvitations?.count || userSuggestions?.count);
-
+  const { isLoading, userMemberships, userSuggestions, userInvitations } = useOrganizationListInView();
   const { hidePersonal } = useOrganizationListContext();
+  const hasOrganizationResources = !!(userMemberships?.count || userInvitations?.count || userSuggestions?.count);
+
+  const sessionTasksContext = useContext(SessionTasksContext);
+  if (sessionTasksContext) {
+    return <ForceOrganizationSelectionFlow hasOrganizationResources={hasOrganizationResources} />;
+  }
 
   return (
-    <Card.Root>
-      <Card.Content sx={t => ({ padding: `${t.space.$8} ${t.space.$none} ${t.space.$none}` })}>
-        <Card.Alert sx={t => ({ margin: `${t.space.$none} ${t.space.$5}` })}>{card.error}</Card.Alert>
-        {isLoading && (
-          <Flex
-            direction={'row'}
-            align={'center'}
-            justify={'center'}
-            sx={t => ({
-              height: '100%',
-              minHeight: t.sizes.$60,
-            })}
-          >
-            <Spinner
-              size={'lg'}
-              colorScheme={'primary'}
-              elementDescriptor={descriptors.spinner}
-            />
-          </Flex>
-        )}
-
-        {!isLoading && <OrganizationListFlows showListInitially={!(hidePersonal && !hasAnyData)} />}
-      </Card.Content>
-      <Card.Footer />
-    </Card.Root>
+    <FlowCard>
+      {isLoading ? (
+        <FlowLoadingState />
+      ) : (
+        <OrganizationListFlows showListInitially={!hidePersonal && hasOrganizationResources} />
```

Comment:
I also took the liberty of refactoring the `showListInitially` condition, as it was a double negative and therefore difficult to understand at first glance. 

Renamed `hasAnyData` constant to `hasOrganizationResources` as it better indicates entities such as suggestions, memberships, and invitations.